### ISSUE TITLE
Training mode of instance norm is hard-coded to True

### ISSUE THREAD

**Issue by ronaldomessina** [hyeonseob-nam/Batch-Instance-Normalization#4]
Hi, I have a question about this line:

https://github.com/hyseob/Batch-Instance-Normalization/blob/96b4ff81d6b19279e0133f632fbc82577fc9e190/models/batchinstancenorm.py#L35

Why is the training mode set to True instead of self.training (as in Line 24, for the BatchNorm section)?

Would really appreciate some feedback on this!
